### PR TITLE
scripts: add script for listing closed bugs

### PR DESCRIPTION
**Pull request by nashif**
Script to be used when creating a release. For regular releases this can
be called this way:

$ list_issues.py -f issues.md -s 2019-09-01

Will list all closed issues since september 1st, 2019 and will create a
markdown file with all issues that can be added as is to the release
notes.

### CODE REVIEW

**Comment by dleach02** [comment]
This appears to work as advertised.

Two questions:

1. Why use '-p' for a branch specifier? Why not '-b'?
2. Do we have any guidance on running pylint3 against python code? Might consider a pass to clean up what is reported currently getting a 7.23/10 rating.

**Review by ioannisg — APPROVED** [review]

**Comment by nashif** [comment]
>     1. Why use '-p' for a branch specifier? Why not '-b'?

addressed

>     2\. Do we have any guidance on running pylint3 against python code? Might consider a pass to clean up what is reported currently getting a 7.23/10 rating.

we do have pylint running in CI

File: scripts/release/list_issues.py [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+    args = parser.parse_args()
+
+
+def main():
+    parse_args()
+
+    token = os.environ.get('GH_TOKEN', None)
+    if not token:
+        sys.exit("""Github token not set in evnironment,
```
Comment (aescolar) [inline scripts/release/list_issues.py]: "evnironment" typo :)

**Review by aescolar — APPROVED** [review]
Beautiful, just the typo and the silly CODEOWNER reported issue with its wrong indent

**Pull request merged**